server/mem_agent/venvstacks_manager.py

```
@@ -0,0 +1,394 @@
+        if not manager.is_built() or force_rebuild:
+            logger.info("Building venvstacks (this may take a while)...")
+            if not manager.build_stacks():
+                return False, "Failed to build venvstacks"
+
+        # Export for deployment
+        if not manager.is_exported() or force_rebuild:
+            logger.info("Exporting venvstacks for deployment...")
+            if not manager.export_stacks():
+                return False, "Failed to export venvstacks"
+
+        return True, f"Stacks ready at {manager.export_dir}"
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)